infra/terraform/secure_pipelines/mob_sts_mock_pipeline.tf

```
@@ -0,0 +1,57 @@
+resource "aws_cloudformation_stack" "mob_sts_mock_pipeline" {
+    name = "mob-sts-mock-pipeline"
```

For the backend-api the naming convention was `${stack_name}-pl`, for example `mob-async-backend-pl`. 
I think this was mostly due to restrictions in how long the names could be for some of the resources it creates.Any issues with aligning this name to the existing name? 
```suggestion
    name = "mob-sts-mock-pl"
```